dbld/shell

```diff
@@ -14,4 +14,4 @@ if [ "$OS_PLATFORM" != "devshell" ]; then
 	echo -e "NOTE: the 'devshell' dbld image is more developer friendly, you might want to use that"
 fi
 cd /source
-HISTFILE=/build/.bash_history exec bash
\ No newline at end of file
+debian_chroot="dbld" HOME=/build exec bash
```

Comment:
Minor note, otherwise LGTM!

It won't let me comment on line 13 ``˙if [ "$OS_PLATFORM" != "devshell" ]; then˙``

AFAIK we do not support other platforms as "shell". (Note: We DO support other platforms as build platforms.)
Based on this, either line 13 is not necessary, or debian_chroot is not "wide enough" for other platforms.

Reply:
I regularly invoke shell on other platforms to try platform specific stuff. So we don't "support" that with all the usability stuff that we have in devshell or the ability to regenerate automake files, but it works fine nevertheless and I think its support at this level is important.

But the prompt is not that important, as I mostly use devshell.